app/templates/search/catalogue/index.html

```diff
@@ -3,10 +3,525 @@
 {%- set pageTitle = 'Catalogue' -%}
 
 {% block content %}
-<div class="tna-section">
+
+
+<div class="tna-section search-box">
   <div class="tna-container">
-    <div class="tna-column tna-column--width-2-3 tna-column--width-5-6-medium tna-column--full-small tna-column--full-tiny">
+    <div class="tna-column--full tna-column--full-small tna-column--full-tiny tna-column--full">
```

Comment:
A column always needs a `tna-column` class. We can then add a full width (`tna-column--full`) which is the "default" for all device widths unless you want something different on `--medium`, `--small` or `--tiny`.

```suggestion
    <div class="tna-column tna-column--full">
```

Reply:
I've updated the code as suggested